.github/workflows/ci.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on: push
```

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches: '*'
```

As discussed this might help to make sure we build on PRs and post-merge.

---

I will change it to suggested, makes sense! However, the build for master will probably be moved in a separate file to be along the releases, as this will all happen on push to master.